testdata/catalogs/test-catalog/catalog.yaml

```
@@ -0,0 +1,104 @@
+{
+    "schema": "olm.package",
+    "name": "prometheus",
+    "defaultChannel": "beta"
+}
+{
+    "schema": "olm.channel",
+    "name": "beta",
+    "package": "prometheus",
+    "entries": [
+        {
+            "name": "prometheusoperator.0.47.0"
+        }
+    ]
+}
+{
+    "schema": "olm.bundle",
+    "name": "prometheusoperator.0.47.0",
+    "package": "prometheus",
+    "image": "localhost/testdata/bundles/registry-v1/prometheus-operator:v0.47.0",
```

We should specify a non-latest tag here:
```suggestion
    "image": "localhost/testdata/bundles/registry-v1/prometheus-operator:v0.47.0:e2e",
```